core/src/main/java/dk/alexandra/fresco/framework/network/AsyncNetwork.java

```diff
@@ -144,6 +144,7 @@ public AsyncNetwork(NetworkConfiguration conf, Duration timeout) {
         try {
           SocketChannel channel = SocketChannel.open();
           channel.connect(addr);
+          channel.socket().setTcpNoDelay(true);
```

Comment:
not sure if it makes a difference, but couldn't we set this property before the `connect` call?

Reply:
Yes, you are right. I have changed it.